dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -1435,12 +1436,20 @@ public Identifier getRelatedIdentifier(Contentlet contentlet,String relationship
         return permissionAPI.filterCollection(contentFactory.getRelatedLinks(contentlet), PermissionAPI.PERMISSION_READ, respectFrontendRoles, user);
     }
 
-    @CloseDBIfOpened
+
+    @Deprecated
     @Override
     public List<Contentlet> filterRelatedContent(Contentlet contentlet, Relationship rel,
             User user, boolean respectFrontendRoles, Boolean pullByParent, int limit, int offset,
             String sortBy)
             throws DotDataException, DotSecurityException {
+        return filterRelatedContent(contentlet, rel, user, respectFrontendRoles, pullByParent, limit, offset);
```

Comment:
I'd log a warning here as well, saying this method no longer respects the sort and to use the other method.

Reply:
Done